src/sidebars/favourites/tearout.js

```diff
@@ -16,6 +16,7 @@
 
                         var windowService = window.windowService;
                         var tearoutWindow = windowService.createTearoutWindow(window.name);
+                        var me = {};
```

Comment:
Yeah I have seen it was already there, if you think it makes sense to give it better name, we can do it, otherwise I am VERY happy for you to merge it (as it fixes bug in my feature :D )